db/migrate/20160527093031_limit_os_description.rb

```diff
@@ -0,0 +1,12 @@
+class LimitOsDescription < ActiveRecord::Migration
+  def up
+    Operatingsystem.all.each do |os|
+      os.update_attribute(:description, os.description.truncate(255))
```

Comment:
@dLobatog I think this is no longer addressed since after Tomer's suggestion you select only OSes without empty string hence including those with NULL in description